### ISSUE TITLE
Connect-PnPOnline command is not working along with Export-M365DSCConfiguration in Azure Runbook

### ISSUE THREAD

**Issue by Infy-Ayusi** [pnp/powershell#3377]
### Notice
Many bugs reported are actually related to the PnP Framework which is used behind the scenes. Consider carefully where to report an issue:

We are running a script in Azure Runbook which performs actions-
-Connects to SharePoint site using Connect-PnPOnline 
-Export M365DSC configuration
Script returns error: Could not load type 'Microsoft.Graph.Authentication.AzureIdentityAccessTokenProvider' from assembly 'Microsoft.Graph.Core, Version=1.25.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'

After consulting with M365DSC Github community contributors, we understood it is occurring due to PnP command. 

Kindly suggest on this issue. 

### Reporting an Issue or Missing Feature
Issue

### Expected behavior 
To connect SharePoint site and work with Export-M365DSCConfiguration command

### Actual behavior 
Could not load type 'Microsoft.Graph.Authentication.AzureIdentityAccessTokenProvider' from assembly 'Microsoft.Graph.Core, Version=1.25.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'

### Steps to reproduce behavior
Make Connect-PnPOnline command call before Export DSC command

### What is the version of the Cmdlet module you are running?
1.12.0 with PS 5.1

### Which operating system/environment are you running PnP PowerShell on?
- [ ] Azure Cloud Shell

**Comment by KoenZomers** [comment]
@Infy-Ayusi Any particular reason why you're using such an old PnP PowerShell version? Could you try this with the latest nightly build of PnP PowerShell? @gautamdsheth has recently implemented a fix for clashes between multiple PowerShell modules using different versions of the same file, which seems to be the case at hand here.

**Comment by gautamdsheth** [comment]
Yes, this issue has been fixed with latest nightly builds of PnP PowerShell.
Will move it to discussion.
